example/CMakeLists.txt

```diff
@@ -22,18 +30,22 @@ endfunction()
 example(cfg/printer printer)
 example(cfg/runner runner)
 
-example(cfg/parallel_runner parallel_runner)
-if (DEFINED ENV{TBB_ROOT})
-  set(TBB_ROOT $ENV{TBB_ROOT})
-  include(${TBB_ROOT}/cmake/TBBBuild.cmake)
-  tbb_build(TBB_ROOT ${TBB_ROOT} CONFIG_DIR TBB_DIR MAKE_ARGS stdver="c++17")
-  find_package(TBB)
-  target_link_libraries(boost_ut_parallel_runner ${TBB_IMPORTED_TARGETS})
+if(APPLE OR "${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
+  example(cfg/parallel_runner parallel_runner)
 endif()
 
+# if(DEFINED ENV{TBB_ROOT})
+#   set(TBB_ROOT $ENV{TBB_ROOT})
+#   # FIXME: include could not find load file:
```

Comment:
Note this, it does not work at all?

Reply:
that used to work on travis but somehow got broken recently, not sure why